modules/administration/pages/monitoring.adoc

```diff
@@ -120,14 +117,14 @@ See https://github.com/boynux/squid-exporter.
 
 == Install and Configure Prometheus
 
-Prometheus is installed from a package, and needs configuration before you can use it to gather metrics.
+Prometheus is installed on your monitoring server from a package, and needs configuration before you can use it to gather metrics.
```

Comment:
Here also, the explicit instruction that Salt Client is an example of manager client van be added, given that it (Salt Client) appears further down in the text.